### PR TITLE
refactor alert instance && support sms(web)

### LINKED ISSUE #1818: [Refactor][web] refactor alert instance
### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dinky/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

refactor alert instance in dinky-web module

**note:** will be use card list to implement

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### LINKED ISSUE #2000: [Feature][alert,web] support sms alert
### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dinky/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

support sms alert

### Use case

support sms alert

### Related issues

support sms alert

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Zzm0809**
## Purpose of the pull request

- refactor alert instance
- support sms(web) using: https://wind.kim/doc/start/
  - [x] The front-end code has been completed
  - [ ] The remaining sending methods of the back-end code have not been implemented yet, marked as todo, and will be implemented later

### list
image
image

### create
create

### modify
image
image
image
image
image

### CODE REVIEW

File: dinky-alert/dinky-alert-sms/src/main/java/org/dinky/alert/sms/SmsSender.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import org.slf4j.LoggerFactory;
+
+import cn.hutool.json.JSONUtil;
+
+/** SmsSender todo: https://wind.kim/doc/start/springboot.html */
+public class SmsSender {
+
+    private static final Logger logger = LoggerFactory.getLogger(SmsSender.class);
+    private SupplierConfig configLoader = null;
+    SmsBlend smsSendFactory = null;
```
Comment (zackyoungh) [inline dinky-alert/dinky-alert-sms/src/main/java/org/dinky/alert/sms/SmsSender.java]: 建议使用final

File: dinky-alert/dinky-alert-wechat/pom.xml [hunk 2]
```diff
@@ -41,7 +41,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <scope>provided</scope>
+            <scope>${scope.runtime}</scope>
```
Comment (zackyoungh) [inline dinky-alert/dinky-alert-wechat/pom.xml]: 这个只有在test中引用

**Review by zackyoungh — CHANGES_REQUESTED** [review]

File: dinky-alert/dinky-alert-sms/src/main/java/org/dinky/alert/sms/SmsSender.java (hunk 1 above)
Comment (Zzm0809) [reply dinky-alert/dinky-alert-sms/src/main/java/org/dinky/alert/sms/SmsSender.java]: > 建议使用final

构造中需要根据 厂商类型获取 配置对象 和 发送工厂对象

File: dinky-web/src/pages/RegCenter/Alert/AlertInstance/components/AlertTypeChoose/InstanceForm/Sms/index.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+
+const Sms = (props: any) => {
+  const {values} = props;
+  /**
+   * render form by sms type
+   */
+  const renderFormBySmsType = useCallback((smsType: number) => {
+    switch (smsType) {
+      case 1:
```
Comment (leechor) [inline dinky-web/src/pages/RegCenter/Alert/AlertInstance/components/AlertTypeChoose/InstanceForm/Sms/index.tsx]: here maybe enum type better?

**Review by leechor — COMMENTED** [review]
LGTM

File: dinky-web/src/pages/RegCenter/Alert/AlertInstance/components/AlertTypeChoose/InstanceForm/Sms/index.tsx (hunk 3 above)
Comment (Zzm0809) [reply dinky-web/src/pages/RegCenter/Alert/AlertInstance/components/AlertTypeChoose/InstanceForm/Sms/index.tsx]: OK，i will use enum to support it

**Review by zackyoungh — APPROVED** [review]
LGTM

**Pull request merged**